### ISSUE TITLE
React Native ondevice look-and-feel

### ISSUE THREAD

**Issue by shilman** [storybooks/storybook#6604]
RN on-device should match storybook web UI look and feel:
- [ ] sidebar navigation
- [ ] ondevice addons

**Comment by shilman** [comment]
Egads!! I just released https://github.com/storybooks/storybook/releases/tag/v5.1.0-alpha.39 containing PR storybookjs/storybook#6603 that references this issue. Upgrade today to try it out!

Because it's a pre-release you can find it on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.

**Comment by Gongreg** [comment]
@shilman, is there anything left to do here? Emotion is added.

**Comment by shilman** [comment]
@Gongreg Yeah AFAIK it hasn't actually been styled to match the 5.x design yet.
